db/migrate/20191002105112_create_versioned_school_certifying_officials.rb

```diff
@@ -0,0 +1,17 @@
+class CreateVersionedSchoolCertifyingOfficials < ActiveRecord::Migration
```

Comment:
Please extract the database changes to their own PR.
We'll get get the migrations deployed and executed first.
Then we can deploy the rest of the code in this PR since it depends on those database changes having already occurred.